torch/onnx/_internal/fx/export.py

```diff
@@ -470,6 +470,7 @@ def export(
 
 def export_without_kwargs(
     fn: Union[torch.nn.Module, Callable],
+    opset_version,
```

Comment:
Possible to make it optional and keyword only? so

fn, *args, ...

Reply:
out of scope for this PR, but I'd love to hear your opinion on this. Which would you prefer?

* optional, default to some default (same behavior as current exporter)
* optional, default to latest
* required
* specified via a separate 'config.py' file (what dynamo and inductor does)
* not configurable